content/tutorials/Team8Tutorial.md

```diff
@@ -0,0 +1,180 @@
+
+This tutorial will teach readers how to use the MPU 6000 to measure the x y and z axises to keep upright
+
+### Learning Objectives
+
+- MPU schematics and ways to utilize the chip
+- Schematic making, as well as using KiCad to 3d render 
+- Arduino C
+
+### Background Information
+
+The MPU chip is primarily used in drones to keep their orientation stable, in your phones, or even in your controllers if you play on it. So, specifically we are trying to do it for drone purposes. We would first need to create the board creating a schematic, then further it by connecting that to a microcontroller that receives the data.
+
+What we're aiming for
+1. Making sure that our wiring can power the device on.
+2. Provides real-time feedback on where exactly the device is orientated.
+
+
+Key Concepts:
+
+1. Circuit Basics
+2. KiCad Basics
+3. Arduino Basics
+
+## Getting Started
+
+You will receive a MPU 6000 Chip that can connect to Arduino IDE
+
+You will also receive a capacitors, wires, and a LTI1962-3.3 voltage regulators. These will be important and maintaing stable voltage throughout the ciruit.
+
+We will be using the Arduino IDE (Integrated Development Environment). This is a free, open source program that allows users to write code and upload it to boards. 
+
+### Required Downloads and Installations
+
+We are going to use KiCad, so if you don't have it installed, click on either KiCad. Now that we have it installed, below is a list of important components.
+
+### Required Components
+
+List your required hardware components and the quantities here.
+
+| Component Name | Quanitity |
+| -------------- | --------- |
+| MPU-6000 module   |  1     |
+| LT1962-3.3 voltage regulator | 1  |
+|Capacitor (1µF)|     1     |
+| Capacitor (.01µF)  |     1     |
+| Capacitor (10µF)|     1     |
+|Capacitor (20µF) |     1     |
+
+
+### Required Tools and Equipment
+
+Computer, Arduino IDE, KiCad
+
+## Part 01: Setting up the Circuit
+
+### Introduction
+
+In this section, we will be discussing where each part should be wired to.
+The data sheet that is attached is the MPU 6000 [LINK](https://invensense.tdk.com/wp-content/uploads/2015/02/MPU-6000-Datasheet1.pdf). 
+
+### Objective
+
+The data sheet provides some insight on what attaches where. You might have noticed that 
+The technical skills learned/needed
+in this challenge. There is no need to go into detail as a
+separation document should be prepared to explain more in depth
+about the technical skills.
+
+### Components
+
+| Component Name | Quanitity |
+| -------------- | --------- |
+| MPU-6000 module   |  1     |
+| LT1962-3.3 voltage regulator | 1  |
+|Capacitor (1µF)|     1      |
+| Capacitor (.01µF)  |     1     |
+| Capacitor (10µF)|     1     |
+|Capacitor (20µF) |     1     |
+
+### Instructional
+
+Assemble the circuit as shown in the below image. 
+![below image](https://github.com/rsevidal/docs/blob/main/content/tutorials/Team13Photos/Screen%20Shot%202024-12-04%20at%204.59.05%20PM.png)
```

Comment:
These images don't exist... they are located in the `Team13Photos` folder which you can refer to relatively like: `Team13Photos/{img name}.png`.

Aren't you team 8? Why is team 13 involved?